deno.json

```
@@ -5,7 +5,8 @@
     "fmt:check": "deno fmt --unstable --check",
     "lint": "deno lint --unstable",
     "build": "deno run -A --unstable script/build.ts",
-    "run": "deno task build && export PLUGIN_URL=\"./build/\" && deno run -A --unstable"
+    "run": "deno task build && export PLUGIN_URL=\"./build/\" && deno run -A --unstable",
+    "fast-run": "export PLUGIN_URL=\"./build/\" && deno run -A --unstable"
```

To match the rest of the tasks:

```suggestion
    "run:fast": "export PLUGIN_URL=\"./build/\" && deno run -A --unstable"
```